hail/python/hail/methods/qc.py

```diff
@@ -492,13 +512,548 @@ def n_discordant(counter):
     return glob, per_sample.cols(), per_variant.rows()
 
 
+class BaseVEPConfig(abc.ABC):
```

Comment:
Just call it `VEPConfig`, I don't think `Base` conveys useful information to the *user* (although it might mean something to us).